custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -34,11 +35,11 @@
     // disable otel runtime-metrics instrumentation; we use micrometer metrics instead
     config.put("otel.instrumentation.runtime-metrics.enabled", "false");
 
-    config.put("otel.traces.exporter", "jaeger-thrift-splunk");
+    config.put("otel.traces.exporter", "otlp");
     // http://localhost:9080/v1/trace is the default endpoint for SmartAgent
     // http://localhost:14268/api/traces is the default endpoint for otel-collector
     config.put(OTEL_EXPORTER_JAEGER_ENDPOINT, "http://localhost:9080/v1/trace");
```

Comment:
If we use otlp then we should not set this property, right?

Reply:
This is just the default value, it should always be set.